extensions/wikia/AchievementsII/css/oasis.scss

```diff
@@ -159,15 +152,12 @@
 			@include clearfix;
 			li {
 				display: inline-block;
-				border-bottom: none!important;
+				border-bottom: none;
```

Comment:
http://stackoverflow.com/questions/2922909/should-i-use-border-none-or-border-0 :)